### ISSUE TITLE
no-consecutive-blank-lines rule doesn't catch lines with whitespace

### ISSUE THREAD

**Issue by chrismbarr** [palantir/tslint#1249]
### Bug Report

- __TSLint version__: `1.10.1`
- __TypeScript version__: `1.8.10`
- __Running TSLint via__: `gulp-tslint` & Visual Studio

__Config:__ `"no-consecutive-blank-lines": true,`

Consider `~~~>` to be a tab and `-` to be a space

#### TypeScript code being linted

```ts
----var x = "foo";
~~~>
~~~>
----
----
~~~>var y = "bar";
```

----

#### Actual behavior

No errors are thrown for consecutive lines that contain space or tab characters, even those for all practical purposes these lines should probably still be considered "blank"

#### Expected behavior

Throw error when there are consecutive blank lines, even if those "blank" lines only contain whitespace.

**Comment by chrismbarr** [comment]
I am attempting to make a PR for this, but the [tests file for this one is really confusing me](https://github.com/palantir/tslint/blob/master/test/rules/no-consecutive-blank-lines/test.ts.lint).  I added some whitespace to some of the blank lines, and now the tests fail as expected, I'm attempting to rewrite the rule in a different way but I'm a little lost.

Where it has `this.createFailure(scanner.getStartPos(), 1, Rule.FAILURE_STRING);` what does `scanner.getStartPos()` return?  I had always assumed it was the line number, but it seems to return the character position from the start of the file.  Is this correct?
[See  :page_facing_up:  In context](https://github.com/palantir/tslint/blob/master/src/rules/noConsecutiveBlankLinesRule.ts#L48)
